django/middleware/csrf.py

```diff
@@ -19,8 +18,10 @@
 from django.utils.functional import cached_property
 from django.utils.http import is_same_domain
 from django.utils.log import log_response
+from django.utils.regex_helper import _lazy_re_compile
 
 logger = logging.getLogger('django.security.csrf')
+token_re = _lazy_re_compile('[^a-zA-Z0-9]')
```

Comment:
I think it would be helpful if this were instead named `invalid_token_re`. The reason is that I coincidentally happened to be reading `csrf.py` and was confused by these lines:
https://github.com/django/django/blob/b746596f5f0e1fcac791b0f7c8bfc3d69dfef2ff/django/middleware/csrf.py#L111-L112
The reason this was confusing is that this isn't a regex that matches tokens. It matches invalid tokens. And then I saw this was changed in this PR only a few days ago.

Reply:
Good catch :dart: , I missed this :facepalm:. Please feel-free to send a patch

Reply:
PR filed here: https://github.com/django/django/pull/14461